lib/parse.js

```diff
@@ -1589,7 +1604,12 @@ function parse($TEXT, options) {
                         }));
                     }
                 }
-                if (is("operator", "=")) {
+                if (is_expand) {
+                    if (!is("punc", "}")) {
+                        token_error(S.token, "Rest element must be last element");
```

Comment:
On second thought, shouldn't this be just `croak("Rest element must be last element")`?